src/services/completions.ts

```diff
@@ -1529,6 +1537,19 @@ namespace ts.Completions {
             });
         }
 
+        function getVariableDeclaration(property: Node): VariableDeclaration | undefined {
+            const variableDeclaration = findAncestor(property, node =>
+                isFunctionBlock(node) || isArrowFunctionBody(node) || isBindingPattern(node)
+                    ? "quit"
+                    : isVariableDeclaration(node));
+
+            return variableDeclaration as VariableDeclaration;

```

Comment:
shouldn't this be `as VariableDeclaration | undefined`?

Reply:
Yes. You are right.